packages/service/ni_measurement_plugin_sdk_service/discovery/_client.py

```diff
@@ -206,7 +206,9 @@ def unregister_service(self, registration_id: str = "") -> bool:
             _logger.exception("Error in unregistering with discovery service.")
             raise
 
-    def resolve_service(self, provided_interface: str, service_class: str = "") -> ServiceLocation:
+    def resolve_service(
+        self, provided_interface: str, service_class: str = "", version: str = ""
```

Comment:
Skipping deployment_target means that if we add it later, the parameters are out of order compared to the .proto file.

https://github.com/ni/ni-apis/blob/main/ni/measurementlink/discovery/v1/discovery_service.proto#L173